src/Hangfire.PostgreSql/Scripts/Install.v23.sql

```diff
@@ -0,0 +1,13 @@
+﻿SET search_path = 'hangfire';
+
+DO $$
+BEGIN
+    IF EXISTS(SELECT 1 FROM "schema" WHERE "version"::integer >= 23) THEN
+        RAISE EXCEPTION 'version-already-applied';
+END IF;
+END $$;
+
+DROP INDEX IF EXISTS ix_hangfire_job_statename_is_not_null;
+CREATE INDEX IF NOT EXISTS ix_hangfire_job_statename_is_not_null ON job USING btree(statename) INCLUDE (id) WHERE statename IS NOT NULL;
```

Comment:
`IF NOT EXISTS` isn't really needed when doing a single script and you drop it one statement before.

Reply:
You're right, fixed now.